README.md

```diff
@@ -58,17 +63,20 @@ $
 
 Use `./hacks/ops-pod --help` to check what other options are available
 
+## Building ops-toolbelt images
 
-### Building ops-toolbelt images
 Dockerfiles for the images are generated from files in the `dockerfile-config` directory.
 
 To build all pre-configured images run:
+
```

Comment:
Please update the sentence above 'To build all pre-configured images run:', as it is no longer accurate. Previously, we built a matrix of images with different cloud provider CLIs, but we no longer do this. Now, we only build one image.

Reply:
Ah right. Thanks for the suggestion!

Reply:
Addressed in [0102cad](https://github.com/gardener/ops-toolbelt/pull/142/commits/0102cad035b1842c8cb187e8f670ff6aa5a7cf45).